src/Core/AndroidNative/maui/src/main/java/com/microsoft/maui/glide/MauiCustomViewTarget.java

```diff
@@ -17,25 +17,32 @@
 public class MauiCustomViewTarget extends CustomViewTarget<ImageView, Drawable> {
     private final ImageLoaderCallback callback;
     private boolean completed;
+    private boolean cleared;
 
     public MauiCustomViewTarget(@NonNull ImageView view, ImageLoaderCallback callback) {
         super(view);
 
         this.completed = false;
+        this.cleared = false;
```

Comment:
You could define the default value on line 20 like you would in C#.

Reply:
Actually, given the default value is `false` also in Java there's no need to set it again.
So I've addressed this also for `completed` field.